tests/__parser__/test_translator.py

```diff
@@ -0,0 +1,126 @@
+from validator.__parser__.translator import Translator
+from validator import rules as R
+
+r = range
```

Comment:
can we please remove aliases for keywords? It might be easier for you but for someone who will later write more tests, might not have need for this. So let's all stay on the same page shall we?

Reply:
Deal! Will change right away